pkgs/servers/softether/4.29.nix

```
@@ -0,0 +1,228 @@
+          src/Cedar/UT.c \
+          src/Cedar/UT.h \
+          src/Cedar/VG.c \
+          src/Cedar/VG.h \
+          src/Cedar/Virtual.c \
+          src/Cedar/Virtual.h \
+          src/Cedar/VLan.c \
+          src/Cedar/VLan.h \
+          src/Cedar/VLanUnix.c \
+          src/Cedar/VLanUnix.h \
+          src/Cedar/VLanWin32.c \
+          src/Cedar/VLanWin32.h \
+          src/Cedar/WaterMark.c \
+          src/Cedar/WaterMark.h \
+          src/Cedar/WebUI.c \
+          src/Cedar/WebUI.h \
+          src/Cedar/Win32Com.cpp \
+          src/Cedar/Win32Com.h \
+          src/Cedar/WinJumpList.cpp \
+          src/Cedar/WinUi.c \
+          src/Cedar/WinUi.h \
+          src/Cedar/Wpc.c \
+          src/Cedar/Wpc.h \
+          src/GlobalConst.h \
+          src/hamcorebuilder/hamcorebuilder.c \
+          src/Mayaqua/Cfg.c \
+          src/Mayaqua/Cfg.h \
+          src/Mayaqua/Encrypt.c \
+          src/Mayaqua/Encrypt.h \
+          src/Mayaqua/FileIO.c \
+          src/Mayaqua/FileIO.h \
+          src/Mayaqua/Internat.c \
+          src/Mayaqua/Internat.h \
+          src/Mayaqua/Kernel.c \
+          src/Mayaqua/Kernel.h \
+          src/Mayaqua/Mayaqua.c \
+          src/Mayaqua/Mayaqua.h \
+          src/Mayaqua/MayaType.h \
+          src/Mayaqua/Memory.c \
+          src/Mayaqua/Memory.h \
+          src/Mayaqua/Microsoft.c \
+          src/Mayaqua/Microsoft.h \
+          src/Mayaqua/Network.c \
+          src/Mayaqua/Network.h \
+          src/Mayaqua/Object.c \
+          src/Mayaqua/Object.h \
+          src/Mayaqua/OS.c \
+          src/Mayaqua/OS.h \
+          src/Mayaqua/Pack.c \
+          src/Mayaqua/Pack.h \
+          src/Mayaqua/Secure.c \
+          src/Mayaqua/Secure.h \
+          src/Mayaqua/Str.c \
+          src/Mayaqua/Str.h \
+          src/Mayaqua/Table.c \
+          src/Mayaqua/Table.h \
+          src/Mayaqua/TcpIp.c \
+          src/Mayaqua/TcpIp.h \
+          src/Mayaqua/Tick64.c \
+          src/Mayaqua/Tick64.h \
+          src/Mayaqua/Tracking.c \
+          src/Mayaqua/Tracking.h \
+          src/Mayaqua/Unix.c \
+          src/Mayaqua/Unix.h \
+          src/Mayaqua/Win32.c \
+          src/Mayaqua/Win32.h \
+          src/vpnbridge/vpnbridge.c \
+          src/vpnclient/vpncsvc.c \
+          src/vpncmd/vpncmd.c \
+          src/vpncmdsys/vpncmdsys.c \
+          src/vpncmdsys/vpncmdsys.h \
+          src/vpncmgr/vpncmgr.c \
+          src/vpndrvinst/vpndrvinst.c \
+          src/vpndrvinst/vpndrvinst.h \
+          src/vpninstall/vpninstall.c \
+          src/vpninstall/vpninstall.h \
+          src/vpnserver/vpnserver.c \
+          src/vpnsetup/vpnsetup.c \
+      ; do
+        sed -i \
+            -e "/^Copyright/s|^Copyright|// Copyright|g" \
+            $f
+      done
+  '';
+
+  meta = with stdenv.lib; {
+    description = "An Open-Source Free Cross-platform Multi-protocol VPN Program";
+    homepage = https://www.softether.org/;
+    license = licenses.gpl2;
```

```suggestion
    license = licenses.asl20;
```
Since https://www.softether.org/5-download/src/190121_switch_to_apache_license
and https://github.com/SoftEtherVPN/SoftEtherVPN_Stable/commit/0e317124ae6e266d8d5c8c5e6b208633b7706af2#diff-9879d6db96fd29134fc802214163b95a